saskatoon/sitebase/static/css/style.css

```diff
@@ -56,19 +56,24 @@
 /*----------------------------------------*/
 html, body {font-family: 'Roboto', sans-serif;font-weight:400;}
 body {
-    min-height: 100vh;
     margin: 0;
     position: relative;
-    padding-bottom: 100px; /* W! footer height */
     box-sizing: border-box;
     background:#fff;
 }
+.main-body {
+    min-height: calc(100vh - 95px); /* wrt. footer height */
```

Comment:
how did you determine the `95px` value
![screenshot_2025-06-17_21:22:26](https://github.com/user-attachments/assets/2b6f4d7a-ba09-4331-8f4d-f020bb30de18)
 ?
I see some whitespace between the bottom of the footer and the bottom of the page. If I change this value for `90px` it disappears

Reply:
Looks like 95px was working when you had still `padding-top: 20px` instead of `padding-top: 15px` in the `footer-copyright-area` 
Either way if there's a formula let's add it as a comment